### PR TITLE
RichCommand: Layout Sort Group

### PR DESCRIPTION
**Pull request by cinqmilleans**
**Resolved / Related Issues**
Add commands Layout Sort Group.

When the option "Sync layout and sorting preferences across directories", the adaptive layout is disable in toolbar but not in context menu. It's fixed.

The layout context menu replaces icons by toggle indicator. This lets you know which layout is active.

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

### CODE REVIEW

**Comment by yaira2** [comment]
Can you group related actions together instead of creating separate files for each one?

**Comment by cinqmilleans** [comment]
It's done.

File: src/Files.App/Actions/Display/LayoutAction.cs [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+				SetProperty(ref isOn, Context.LayoutType == LayoutType, nameof(IsOn));
+
+			if (e.PropertyName is not null)
+				OnContextChanged(e.PropertyName);
+		}
+
+		protected virtual void OnContextChanged(string propertyName) { }
+	}
+
+	internal class LayoutPreviousAction : IAction
```
Comment (yaira2) [inline src/Files.App/Actions/Display/LayoutAction.cs:130]: What does this do?

Comment (cinqmilleans) [reply src/Files.App/Actions/Display/LayoutAction.cs:130]: It is a command that allows you to navigate between all the layouts. This is not used but may be available in the future command handler or with a new or custom hotkey.

Comment (yaira2) [reply src/Files.App/Actions/Display/LayoutAction.cs:130]: We already have `ctrl` + `+` and `ctrl` + `-`.

**Comment by yaira2** [comment]
> The layout context menu replaces icons by toggle indicator. This lets you know which layout is active.

Doesn't MenuItem have support for icon as well?

**Comment by cinqmilleans** [comment]
Yes, but only a fonticon, not an opacityicon. The columns layout uses opacityicon. I tested with the tool icon + font icon, it's not aesthetic. I'll send a screenshot later.

File: src/Files.App/Actions/Display/LayoutAction.cs (hunk 1 above)
Comment (cinqmilleans) [reply src/Files.App/Actions/Display/LayoutAction.cs]: I replaced Previous/Next with DecreaseSize/IncreaseSize.

**Comment by cinqmilleans** [comment]
Here is a screenshot. We can find a trick for the image of columns.
Layout

**Comment by yaira2** [comment]
I'm going to be updating the other icons in the flyout as well, do you have any ideas on fixing the issue?

**Comment by cinqmilleans** [comment]
It will probably be necessary to create a UserControl derived from ToggleMenuFlyoutItem (as it was done for the buttons). In the preview, the Columns icon is already missing in this context menu. I propose to deal with this problem in a future pr. I will commit the code with the check + icon.

**Comment by cinqmilleans** [comment]
It's done.

**Comment by yaira2** [comment]
@cinqmilleans can you fix the merge conflict?

**Comment by cinqmilleans** [comment]
It's done.

File: src/Files.App/Strings/en-US/Resources.resw [hunk 2]
```diff
@@ -1419,7 +1419,7 @@
   <data name="DateModifiedLowerCase" xml:space="preserve">
     <value>Date modified</value>
   </data>
-  <data name="NavToolbarArrangementOptionOriginalFolder.Text" xml:space="preserve">
```
Comment (yaira2) [inline src/Files.App/Strings/en-US/Resources.resw:1422]: Can you search to see if this is still being used? Same for the other strings that were removed.

Comment (cinqmilleans) [reply src/Files.App/Strings/en-US/Resources.resw:1422]: I had already checked and I just checked again.

**Pull request merged**